### ISSUE TITLE
[BUG] SteamVR can't start on a fresh install: Error 109 - Not initialized

### ISSUE THREAD

**Issue by anxest** [ValveSoftware/SteamVR-for-Linux#696]
After formatting my computer (to fix other issues) and reinstalling SteamVR, I can't start SteamVR anymore. A window displays "Not initialized (109)".

**Describe the bug**
The error message is displayed once or twice and then, the steamVR app in steam automatically closes, on most attempts. Once in a while, it will stay opened. The steamVR dock will then display on screen with extra messages: 
* Some add-ons blocked: Due to a recent crash, some SteamVR add-ons have been Blocked
* SteamVR Fail: A key component of SteamVR isn't working properly. Please quit SteamVR, then visit Task Manager and End any remaining SteamVR tasks before restarting the app (307)

Various windows will also display:
* SteamVR Settings will remain empty
* No title window empty as well
* No title window showing the settings allowing to manage the add-ons

The headset will remain turned off and never display anything. 

**To Reproduce**
1. Go to SteamVR app on steam
2. Click on Start
4. See error 109

**Expected behavior**
SteamVR displaying on HMD, without errors, allowing to process the room setup

**System Information (please complete the following information):**
* Steam client version (build number or date): steam-runtime_0.20240415.84615 (latest beta)
* SteamVR version: beta 2.5.4 (latest beta)
* Distribution (e.g. Ubuntu): Archlinux with GnomeShell (X.org and AMD GPU)
* Steam runtime diagnostics: [Steam runtime diagnostics.txt](https://gist.github.com/anxest/ede40f34a22080518df00d6666bf3229)
* Steam and SteamVR logs: [steam-logs.tar.gz](https://github.com/ValveSoftware/SteamVR-for-Linux/files/15262613/steam-logs.tar.gz)
* Minidumps: [steam-minidumps.tar.gz](https://github.com/ValveSoftware/SteamVR-for-Linux/files/15262629/steam-minidumps.tar.gz)

**Screenshots**
![Capture d’écran du 2024-05-09 14-20-47](https://github.com/ValveSoftware/SteamVR-for-Linux/assets/19307400/a9d4c556-2a42-40a2-a946-6668754c506c)

**Additional context**
Same behavior with steam client in latest beta or latest non-beta, and with various versions of steamVR (non-beta, latest beta, previous version), steam-runtime and unofficial steam-native-runtime
Following the instructions to "end any remaining steamVR tasks before restarting" won't do any visible change.
All other non-vr games I tried started correctly and allowed me to play without trouble.
I didn't manage to perform the room setup so far.

**Comment by kisak-valve** [comment]
Hello @anxest, your system is configured to run 64 bit Vulkan applications with AMDVLK. Can you test how SteamVR behaves with mesa/RADV?

**Comment by anxest** [comment]
Thank you, I was able to perform a room setup and starting some games by launching steam with mesa (`VK_DRIVER_FILES=/usr/share/vulkan/icd.d/radeon_icd.i686.json:/usr/share/vulkan/icd.d/radeon_icd.x86_64.json steam`) and then again after uninstalling amdlvk and lib32-amdvlk.

The error 109 is gone too, but I still have the weird untitled windows displaying, and the "Some add-ons blocked" messages displaying at each start. 
Plus, when I try to turn off steamVR from the headset, it doesn't and a new gnome window displays that there is no available app to open "vrmonitor://quit". I guess I missed to install a pkg for that. It closes correctly when I exit from the steamvr dock on screen though, but without turning the bases off.

Not sure if I should register other tickets or just leave this one open for those minor issues.

Edit: I simply had to activate the powercontrol option to turn off the bases, so that's not an issue anymore.
